src/node/in_memory.rs

```diff
@@ -1377,6 +1374,7 @@ impl<S: ForkSource + std::fmt::Debug + Clone + Send + Sync + 'static> InMemoryNo
         tracing::info!("│   TRANSACTION SUMMARY   │");
         tracing::info!("└─────────────────────────┘");
 
+        dbg!(&tx_result.result);
```

Comment:
It should not be a problem in release but if it we do not need this statement we should remove it